.env.dev

```diff
@@ -1,33 +1,43 @@
 ## General ##
 #############
 
-COMPOSE_FILE=docker-compose.yml:docker-compose.prod.yml
 COMPOSE_PROJECT_NAME=podkrepi-dev
-NODE_ENV=development # development, production
+COMPOSE_FILE=docker-compose.yml:docker-compose.prod.yml
+
+WEB_DEPLOY_IMAGE=ghcr.io/daritelska-platforma/frontend/nightly/web
+API_DEPLOY_IMAGE=ghcr.io/daritelska-platforma/frontend/nightly/api
+FLYWAY_DEPLOY_IMAGE=ghcr.io/daritelska-platforma/frontend/nightly/flyway
+
+DEPLOY_TAG=
+
+APP_ENV=production # development, staging, production
```

Comment:
Maybe we need to add here the actual environment like `nightly` so we can differentiate that in Senty release for example